lib/static/components/modals/screenshot-accepter/index.jsx

```diff
@@ -208,13 +208,13 @@ class ScreenshotAccepter extends Component {
         return imageId;
     }
 
-    _preloadAdjacentImages(offset = PRELOAD_IMAGE_COUNT) {
-        const screensCount = size(this.state.stateNameImageIds);
-        const previosImagesIndex = (screensCount + this.state.activeImageIndex - offset) % screensCount;
-        const nextImagesIndex = (this.state.activeImageIndex + offset) % screensCount;
+    _preloadAdjacentImages(activeImageIndex, stateNameImageIds, offset = PRELOAD_IMAGE_COUNT) {
```

Comment:
Why does it need to be changed?

Reply:
Because it wasn't quite right before. The method used state in it, and was called right after updating the state. But state is not *guaranteed* to be updated after the `setState` line. It may happen later. This became even more apparent in tests.